### DISCUSSION TITLE
Support for Laravel 8?

### DISCUSSION THREAD

**Discussion by FranOvi** [qruto/laravel-wave#23] [Q&A] [answered]
Hi! I'm working on a project with Laravel 8.78, but it seems that you require illuminate libraries from laravel 9. I see that you changed php requirement, and don't see mention of minimum Laravel version in the documentation, so, is there any chance it could be changed?.

**Comment by slavarazum** [comment, marked as answer]
Hello there! 👋

I wanted to bring to your attention that Laravel 8 is now considered an outdated version. You can refer to the [Support Policy](https://laravel.com/docs/10.x/releases#support-policy) for more details. 

I highly recommend upgrading to a newer supported version, as it will enhance the security of your application and provide access to the latest features.

To simplify the upgrade process, you might want to consider utilizing [Laravel Shift](https://laravelshift.com/). It is an excellent automated tool specifically designed for Laravel upgrades.

**Comment by FranOvi** [comment]
Thanks for the suggestion! I understand, but this is a project we are actively working on, and just did a big migration to this Laravel version last year and it is not in our plans to do another one just yet.

**Reply by slavarazum** [reply]
It's understandable that considerable resources were likely expended in transitioning to version 8. Nevertheless, to guarantee the stable evolution of your project, it's advisable to employ the latest versions. I've discovered that Laravel Shift offers an automated migration process from version 8 to version 10, priced at just $48. For an actively developing project, this is a relatively minor expense considering the substantial benefits that are likely to be realized as a result. Stay upgraded, enjoy it 😉
